.github/actions/azureml-test/action.yml

```diff
@@ -15,9 +15,12 @@ inputs:
   TEST_KIND:
     required: true
```

Comment:
New method that would not require a VM: https://learn.microsoft.com/en-us/azure/developer/github/connect-from-azure-openid-connect